custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/2-line_cards/card_cover_with_buttons.yaml

```diff
@@ -35,6 +35,11 @@ card_cover_with_buttons:
             card:
               type: "custom:button-card"
               template: "widget_icon"
+              state:
+                - value: 'closed'
+                  styles:
+                    icon:
+                      - color: rgba(var(--color-theme),0.4)
```

Comment:
You have accidentally added the changes for another pr and this must be resolved to fix the merge conflict.